code/nebula/neb.cpp

```diff
@@ -288,6 +288,16 @@ void parse_nebula_table(const char* filename)
 					poofp->density = 1 / (poofp->density * poofp->density * poofp->density);
 				}
 
+				if (optional_string("$Alignment:")) {
+					SCP_string type;
+					stuff_string(type, F_NAME);
+
+					if (!stricmp(type.c_str(), " VERTICAL"))
```

Comment:
The `stuff_string` function should skip over any whitespace so the leading space before VERTICAL wouldn't be detected.